lib/modules/platform/bitbucket-server/readme.md

```diff
@@ -49,7 +49,7 @@ Once it's running and initialized, the quickest way to testing with Renovate is:
 At this point you should have a project ready for Renovate, and the `@renovate-bot` account ready to run on it. You can then run like this:
 
 ```
-npx renovate --platform=bitbucket-server --endpoint=http://localhost:7990 --git-fs=http --username=renovate-bot --password=abc123456789! --log-level=debug --autodiscover=true
+pnpm start --platform=bitbucket-server --endpoint=http://localhost:7990 --git-fs=http --username=renovate-bot --password=abc123456789! --log-level=debug --autodiscover=true
```

Comment:
These are very different. npx works for anyone who has Node.js installed. "pnpm start" only works if you (a) have cloned the repo source code, and (b) have pnpm, and (c) have already run pnpm install. 

I suggest reverting these changes